src/main/java/racingcar/model/Cars.java

```diff
@@ -1,22 +1,115 @@
 package racingcar.model;
 
+import racingcar.view.ResultView;
+
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
+import java.util.Random;
 
 public class Cars {
+    public final int RANDOM_RANGE = 10;
 
     private List<Car> cars;
 
     public Cars() {
         cars = new ArrayList<>();
     }
 
+    public Cars(int number) {
+        this();
+        this.setCarNumber(number);
+    }
+
+    public Cars(String[] carNames){
+        this();
+        this.setCarNames(carNames);
+    }
+
+
     public void setCarNumber(int number) {
         for (int i = 0; i < number; i++) {
             cars.add(new Car());
         }
     }
+
+    public void setCarNames(String[] carNames) {
+        for (String carName : carNames) {
+            this.addCar(carName);
+        }
+    }
+
+    public void addCar(String name) {
+        Car newCar = new Car(name);
+        cars.add(newCar);
+    }
+
+    // List 에 영향을 못 미치게 해주기
+    // 수정부가능한 컬렉션 반환
     public List<Car> getCar() {
-        return cars;
+        return Collections.unmodifiableList(this.cars);
+    }
+
+    public int getCatNumber() {
+        return cars.size();
+    }
+
+    public String getWinners() {
+        return getWinnersNames(calculateMaxPosition(cars));
+    }
+
+    private String getWinnersNames(int max) {
+        StringBuilder winners = new StringBuilder();
+        for (Car car : cars) {
+            boolean equal  = checkEqualMax(max, car);
+
+            if (equal){
```

Comment:
객체에게 포지션이 같은지 물어보세요! 
https://velog.io/@reasonz/%EC%9E%90%EB%B0%94%EC%8A%A4%ED%81%AC%EB%A6%BD%ED%8A%B8-%EA%B0%9D%EC%B2%B4%EC%8A%A4%EB%9F%BD%EA%B2%8C-%EC%82%AC%EC%9A%A9%ED%95%98%EA%B8%B0

```suggestion
            if (car.equalsPosition(max)){
```